packages/cli/src/commands/generate.ts

```diff
@@ -173,18 +173,27 @@ export class Generate extends Command {
     }
   }
 
-  getModelNameAndContext(modelName: string, context?: string): {model: string; context: string} {
+  getModelNameAndContext(modelName: string, context?: string): {model: string; context?: string} {
     const modelSegments = modelName.split(/[\\/]/)
-    const contextSegments = (context || '').split(/[\\/]/)
+
     if (modelSegments.length > 1) {
       return {
         model: modelSegments[modelSegments.length - 1],
         context: path.join(...modelSegments.slice(0, modelSegments.length - 1)),
       }
     }
+
+    if (context !== undefined && context !== null && context !== '') {
```

Comment:
This is good if you're happy with this solution! Two other options that are a little shorter:

1. `if (context != null && context !== '')`
2. `if (Boolean(context)) (context as string).split(/[\\/]/)`

We can either take advantage of `!= null` checking for both `null` and `undefined` but no other falsy values, or the fact that we're a little smarter than the TS compiler and just do a manual cast to `string` since we know the value can't be `undefined` at that point

Reply:
Good point! Happy to go with 2. Done here a8ca8fd